vms/nftfx/mint_operation.go

```diff
@@ -34,13 +34,13 @@ func (op *MintOperation) Cost() (uint64, error) {
 
 // Outs Returns []TransferOutput as []verify.State
 func (op *MintOperation) Outs() []verify.State {
-	outs := []verify.State{}
-	for _, out := range op.Outputs {
-		outs = append(outs, &TransferOutput{
+	outs := make([]verify.State, len(op.Outputs))
```

Comment:
q: isn't marginally better to preallocate capacity only (`make([]verify.State,0, len(op.Outputs))`) and then append new element? I wonder if `make([]verify.State, len(op.Outputs))` waste time allocating default values